ecosystem/sep-0012.md

```diff
@@ -228,11 +285,9 @@ Status | Description
 -------|------------
 ACCEPTED | The field has been validated. When all required fields are accepted, the [Customer Status](#customer-statuses) should also be accepted.
 PROCESSING | The field is being validated. The client can make [GET /customer](#customer-get) requests to check on the result of this validation in the future.
-NOT_PROVIDED | The field has not been sent by the client. The field is required unless the `optional` property is `true`. The [Customer Status](#customer-statuses) should be `NEEDS_INFO` if a field is required but has not been provided.
```

Comment:
💭 I wonder if it is worth trying to keep `NOT_PROVIDED`, and returning `provided_fields` even when they haven't been provided yet. That would make this change less breaking, although would probably require renaming `provided_fields` to `all_fields` or something more generic.

Reply:
Since we're moving the field-level `status` field to a new object I don't think keeping `NOT_PROVIDED` makes this changes less breaking personally.

Reply:
That's a good point. 👍🏻